tests/integration/test_trading.py

```diff
@@ -294,3 +296,175 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
+    vega.mint(
+        MM_WALLET.name,
+        asset="VOTE",
+        amount=1e4,
+    )
+    vega.forward("10s")
+
+    # Create asset
+    vega.create_asset(
+        MM_WALLET.name,
+        name=ASSET_NAME,
+        symbol=ASSET_NAME,
+        decimals=1,
+        max_faucet_amount=10 * mint_amount * 1e5,
+    )
+    vega.forward("10s")
+    vega.wait_for_total_catchup()
+
+    asset_id = vega.find_asset_id(symbol=ASSET_NAME)
+
+    for wallet in WALLETS:
+        vega.mint(
+            wallet.name,
+            asset=asset_id,
+            amount=mint_amount,
+        )
+    vega.forward("10s")
+    vega.create_simple_market(
+        market_name="CRYPTO:BTCDAI/DEC22",
+        proposal_key=MM_WALLET.name,
+        settlement_asset_id=asset_id,
+        termination_key=TERMINATE_WALLET.name,
+        market_decimals=1,
+    )
+
+    market_id = vega.all_markets()[0].id
+
+    vega.submit_liquidity(
+        key_name=MM_WALLET.name,
+        market_id=market_id,
+        commitment_amount=initial_commitment,
+        fee=0.002,
+        buy_specs=[("PEGGED_REFERENCE_MID", 50, 1)],
+        sell_specs=[("PEGGED_REFERENCE_MID",50, 1)],
+        is_amendment=False,
+    )
+    # Add transactions in the proposed market to pass opening auction at price 1000
+    vega.submit_order(
+        trading_key=PARTY_A.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        price=800,
+        volume=initial_volume,
+    )
+
+    vega.submit_order(
+        trading_key=PARTY_A.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        price=1000,
+        volume=initial_volume,
+    )
+
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=1000,
+        volume=initial_volume,
+    )
+
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=1200,
+        volume=initial_volume,
+    )
+    vega.wait_for_total_catchup()
+    vega.wait_fn(1)
+    #Check order status/ market state 
+    print(vega.get_latest_market_data(market_id=market_id))
+
+    market_info = vega.market_info(market_id=market_id)
+    linear_slippage_factor = float(market_info.linear_slippage_factor)
+    quadratic_slippage_factor = float(market_info.quadratic_slippage_factor)
+    market_data = vega.get_latest_market_data(market_id=market_id)
+    markprice = market_data.mark_price
+
+    estimate_margin_open_vol_only, estimate_liquidation_price_open_vol_only = vega.estimate_position(
+        market_id,
+        open_volume=1,
+        side=["SIDE_SELL", "SIDE_SELL"],
+        price=[1.01, 1.02],
```

Comment:
These prices, sell prices well below the mark price, cause some unexpected behaviour for me. As with the `collateral_available` comment maybe start with the happy case before edge cases.